src/vs/base/common/platform.ts

```diff
@@ -80,7 +80,19 @@ if (typeof navigator === 'object' && !isElectronRenderer) {
 	_isIOS = (_userAgent.indexOf('Macintosh') >= 0 || _userAgent.indexOf('iPad') >= 0 || _userAgent.indexOf('iPhone') >= 0) && !!navigator.maxTouchPoints && navigator.maxTouchPoints > 0;
 	_isLinux = _userAgent.indexOf('Linux') >= 0;
 	_isWeb = true;
-	_locale = navigator.language;
+
+	// Gather loader configuration since that contains the locale
+	let loaderConfiguration: any = null;
+	if (typeof globals.require !== 'undefined' && typeof globals.require.getConfig === 'function') {
+		// Get the configuration from the Monaco AMD Loader
+		loaderConfiguration = globals.require.getConfig();
+	} else if (typeof globals.requirejs !== 'undefined') {
+		// Get the configuration from requirejs
+		loaderConfiguration = globals.requirejs.s.contexts._.config;
+	}
+	const configuredLocale = loaderConfiguration?.['vs/nls']?.['availableLanguages']?.['*'] as string | undefined;
```

Comment:
@alexdima do you know if this is this the best way to get the locale out of the loader configuration?

Reply:
You could export proper API for it via `vs/nls`. Today, `vs/nls` exports a function called `localize`, as well as other functions like `.setPseudoTranslation`, `.create`, `.load`. You could export via proper API from `vs/nls` the current configured locale. If you decide to do it, then please also add a mock implementation for it at `nls.mock.ts`.

Reply:
Ooo this is a great idea!